plutus-scripts/PlutusScripts/Hashing/Common.hs

```diff
@@ -229,6 +253,8 @@ hashingParamsV3 =
     , blake2b_224Short = blake2b_224ShortIO
     , keccak_256Long = keccak256LongIO
     , keccak_256Short = keccak256ShortIO
+    , ripemd_160Short = ripemd160ShortIO
```

Comment:
This adds `ripemd_160` to the existing hashing tests but I'm not sure if they actually get run anywhere so I've added a standalone test in `Ripemd_160.hs` that gets run in `cardano-node-tests`.

Reply:
I'm not sure either, @mkoura do you know where and how this is used?